backend/helpers.py

```diff
@@ -0,0 +1,18 @@
+from backend_config import print_logs
+
+def cull_duplicates(fp):
+    import pandas as pd
```

Comment:
In general it is a good practice to put all imports at the beginning of the file. In this case the `pandas` library should seldom be imported (ideally you download and process the data once and never again) so it might be a good idea to import it only when used.